wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceValidator.java

```diff
@@ -94,12 +85,6 @@ public void validateConfig(WorkspaceConfig config)
           environment.getMachines().entrySet()) {
         validateMachine(machineEntry.getKey(), machineEntry.getValue());
       }
-
-      try {
-        runtimes.validate(environment);
```

Comment:
Java doc should be updated along with these changes https://github.com/eclipse/che/pull/10385/files#diff-5f46ddd221720706a3210208652a0f1eR57

Reply:
done